packages/sdk/src/modules/middlewareModule/utils/getRequestSender.ts

```diff
@@ -86,6 +87,34 @@ export const getRequestSender = (options: Options): RequestSender => {
     };
   };
 
+  const defaultLogger: Logger = {
```

Comment:
We can keep the defaultLogger in a separate file and export it.

Reply:
done [22c9082](https://github.com/vuestorefront/vue-storefront/pull/7227/commits/22c9082e868f2af96c4131d9c7e0ed97f83561c5)